pulsar-client-cpp/README.md

```diff
@@ -130,9 +130,15 @@ brew install openssl
 export OPENSSL_INCLUDE_DIR=/usr/local/opt/openssl/include/
 export OPENSSL_ROOT_DIR=/usr/local/opt/openssl/
 
-# For Protobuf
-brew tap homebrew/versions
-brew install protobuf260 boost boost-python log4cxx jsoncpp
+# For Protobuf 2.6.0
```

Comment:
We actually don't need a specific version of protobuf anymore. We can just leave this at `brew install protobuf`

Reply:
Got it, will update.